sequence_processing_pipeline/templates/nuqc_job.sh

```diff
@@ -93,123 +87,66 @@ function mux-runner () {
             sed -r "1~4s/^@(.*)/@${i}${delimiter}\1/" \
             >> ${seqs_r1} &
-        cat ${r2_filt} | \
-            sed -r "1~4s/^@(.*)/@${i}${delimiter}\1/" \
-            >> ${seqs_r2} &
         cat ${r1_filt} | \
             gzip -c > ${r1_adapter_only} &
-        cat ${r2_filt} | \
-            gzip -c > ${r2_adapter_only} &
         wait
 
         rm ${r1_filt} &
-        rm ${r2_filt} &
         wait
     done
-}
-export -f mux-runner
-
-function minimap2-runner () {
-    if [[ -z ${fcurrent} ]]; then
-        echo "fcurrent not set"
-        exit 1
-    fi
-
-    jobd=${TMPDIR}
-
-    id_map=${jobd}/id_map
-    if [[ ! -f ${id_map} ]]; then
-        echo "No samples..."
-        return
-    fi
-
-    seqs_r1=${jobd}/seqs.r1.fastq
-    seqs_r2=${jobd}/seqs.r2.fastq
-    seqs_mmpe_r1=${jobd}/seqs.mmpe.r1.fastq
-    seqs_mmpe_r2=${jobd}/seqs.mmpe.r2.fastq
-    seqs_mmpese=${jobd}/seqs.mmpese.fastq
-    seqs_mmpese_r1=${jobd}/seqs.mmpese.r1.fastq
-    seqs_mmpese_r2=${jobd}/seqs.mmpese.r2.fastq
-
-    # PE operation
-    minimap2 -2 -ax sr -t 12 \
-        ${fcurrent} \
-        ${seqs_r1} \
-        ${seqs_r2} | \
-            samtools fastq \
-                -@ 4 \
-                -f 12 \
-                -F 256 \
-                -N \
-                -1 ${seqs_mmpe_r1} \
-                -2 ${seqs_mmpe_r2}
-
-    rm ${seqs_r1} &
-    rm ${seqs_r2} &
-    # no need to block
-
-    # SE operation
-    # run r1/r2 serially to avoid minimap2 picking up on interleve
-    minimap2 -2 -ax sr -t 12 \
-        ${fcurrent} \
-        <(cat ${seqs_mmpe_r1} ${seqs_mmpe_r2}) | \
-            samtools fastq \
-                -@ 4 \
-                -f 4 \
-                -F 256 \
-                -0 ${seqs_mmpese}
-
-    rm ${seqs_mmpe_r1} &
-    rm ${seqs_mmpe_r2} &
-    # no need to block
-
-    {{splitter_binary}} ${seqs_mmpese} ${seqs_mmpese_r1} ${delimiter} ${r1_tag} &
-    {{splitter_binary}} ${seqs_mmpese} ${seqs_mmpese_r2} ${delimiter} ${r2_tag} &
-    wait
-
-    rm ${seqs_mmpese} &
 
-    fastq_pair -t 50000000 ${seqs_mmpese_r1} ${seqs_mmpese_r2}
-    rm ${seqs_mmpese_r1}.single.fq &
-    rm ${seqs_mmpese_r2}.single.fq &
-    rm ${seqs_mmpese_r1} &
-    rm ${seqs_mmpese_r2} &
+    # minimap/samtools pair commands are now generated in NuQCJob._generate_mmi_filter_cmds()
+    # and passed to this template. This method assumes ${jobd} is the correct location to
+    # filter files, the initial file is "${jobd}/seqs.r1.fastq"), and the output name is
+    # "${jobd}/seqs.r1.ALIGN.fastq".
+    {{mmi_filter_cmds}}
+
+    {{movi_path}} query \
+        --index /scratch/movi_hg38_chm13_hprc94 \
+        --read ${jobd}/seqs.r1.ALIGN.fastq \
+        --stdout > ${jobd}/seqs.movi.txt
+        
+    cat ${jobd}/seqs.movi.txt | \
+        python /home/qiita/qiita_spots/human_host_filtration/scripts/qiita_filter_pmls.py - | \
+        seqtk subseq ${jobd}/seqs.r1.ALIGN.fastq - > ${jobd}/seqs.r1.final.fastq
+         
```

Comment:
After email discussion, we need to keep the seqs.movi.txt, can you move it to one of the folders that are passed to the final results and stored in Qiita? Maybe worth saving it as seqs.movi.${SLURM_ARRAY_TASK_ID}.txt